### PR TITLE
Paste and erase in Browse Data table

### PR DESCRIPTION
**Pull request by MilosSubotic**
Hi.
I added paste (Ctrl+V) and erase (Delete) functionality for cells in Browse Data table (ExtendedTableWidget).
I add it for my convenience, please consider to merge it if you think that is usefull.

### CODE REVIEW

**Comment by justinclift** [comment]
Awesome. :smile:

@MKleusberg What do you reckon, is this good to merge?

**Comment by justinclift** [comment]
@MKleusberg Ping. :wink:

File: src/ExtendedTableWidget.cpp [hunk 1]
```diff
@@ -64,12 +65,152 @@ void ExtendedTableWidget::copy()
...
+    int selectedColumns = indices.back().column() - firstColumn + 1;
+
+
+    // If not selected only one cell then check does selection match cliboard dimensions
+    if(selectedRows != 1 || selectedColumns != 1)
+    {
+        if(selectedRows != clipboardRows || selectedColumns != clipboardColumns)
+        {
+            // Ask user is it sure about this
+            QMessageBox::StandardButton reply = QMessageBox::question(this, tr("asdf"),
```
Comment (MKleusberg) [inline src/ExtendedTableWidget.cpp]: This shouldbe QApplication::applicationName() instead of tr("asdf")   :wink:

**Comment by MKleusberg** [comment]
@MilosSubotic Thank you very much for contributing :smile: This looks very promising! I'll do some more testing later but I think we can merge this soon :smile: For now there are two more things:
1) Can you maybe squash the 4 commit into one (https://github.com/sqlitebrowser/sqlitebrowser/wiki/Merging-Commits) so we can have a nice commit history?
2) The erase part is superseded by now because I've implemented something similar for issue #443. I didn't see your PR back then :frowning: But I'd still like to merge the pasting code you've added :)

**Comment by MilosSubotic** [comment]
@MKleusberg
- tr("asdf") corrected.
1) Commits-merged.
2) Even better with Del and Alt+Del

**Comment by MKleusberg** [comment]
I had a closer look now and this looks really good :+1: Looks like you've really though of everything, @MilosSubotic . Nice work :smile: If you remove the erase code (or somehow merge it with the existing code from e74645bb1f74395016bfda110dd4d996bb1d5cb1) we can merge this :)

**Comment by MilosSubotic** [comment]
I synced my fork to the newest commit [4727b0b](https://github.com/sqlitebrowser/sqlitebrowser/commit/4727b0bb9226377ca2eb1c46a0ec6f4697b53746).
My erase() is erased.
Commits squashed as you like it.

**Comment by MKleusberg** [comment]
Awesome :smile: Thank you very much :+1:

**Comment by justinclift** [comment]
@MilosSubotic Great timing too.  Our next release will probably be tomorrow, so this code will have a decent amount of real world users pretty soon. :grin:

**Comment by tonypdmtr** [comment]
When pasting, the data is quoted.  Maybe the quotes should be removed, or else there is always need for manual editing after pasting.

**Comment by MilosSubotic** [comment]
I will correct that today.

**Pull request merged**